package.json

```diff
@@ -4,14 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "devStart": "nodemon server.js"
+    "devStart": "nodemon server.js",
+    "server": "npx peerjs -p 9000"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "ejs": "^3.1.5",
     "express": "^4.17.1",
+    "peer": "^0.5.3",
```

Comment:
None of the changes in this file should matter since we are about to delete everything at the root level of the project.